### ISSUE TITLE
memory clock lock on 1660 ti's

### ISSUE THREAD

**Issue by AlienMiner303** [andru-kun/wildrig-multi#288]
cant lock memory clock on my 1660 ti's, works fine on 4070s but 1660 ti's wont work. ive tried 810 and 5000 and miner wont lock memory, any workaround or bug?

**Comment by andru-kun** [comment]
yep, this is known issue, old cards are not supported well. Only on HiveOS/mmpOS it should work, since there miner trying to call internal tools like nvtool.

**Comment by AlienMiner303** [comment]
Ok good to know, thank you for the response
